src/operator/contrib/ctc_loss-inl.h

```diff
@@ -331,21 +333,20 @@ class CTCLossOp : public Operator {
     Stream<xpu> *s = ctx.get_stream<xpu>();
 
     Tensor<xpu, 3, real_t> data_grad =
-        in_grad[ctc_loss::kData].get<xpu, 3, real_t>(s);
+      in_grad[ctc_loss::kData].get<xpu, 3, real_t>(s);
     Tensor<xpu, 1, real_t> output_grad =
-        out_grad[ctc_loss::kOut].get<xpu, 1, real_t>(s);
+      out_grad[ctc_loss::kOut].get<xpu, 1, real_t>(s);
 
     Tensor<xpu, 3, real_t> data_grad_computed =
-        out_data[ctc_loss::kGrad].get<xpu, 3, real_t>(s);
+      out_data[ctc_loss::kGrad].get<xpu, 3, real_t>(s);
```

Comment:
same here.

Reply:
Removed. However, these 4 space indentation seems a violation of Google C++ style guide. Are we ignoring them in the lint?
https://google.github.io/styleguide/cppguide.html